oauth2_provider/models.py

```diff
@@ -171,9 +174,7 @@ class Grant(models.Model):
     pkce_code = models.TextField(null=True)
 
     def check_pkce(self, given_value):
-        if given_value is None:
-            if self.pkce_code is None:
-                return True
+        if given_value is None or self.pkce_code is None:
```

Comment:
changed this because it's now only called if `enable_pkce` is `True`, so None is an invalid value